### DISCUSSION TITLE
Option for promotions and discounts

### DISCUSSION THREAD

**Discussion by hassanyousufx** [inspireui/support#5461] [New Feature/Ideas Requests]
I think it will be nice to add options like Buy 1 Get 1. We have plugins that work on the website but they don't reflect on the cart screen in app or checkout.

1. Buy 1 Get 1
2. Buy X Get Y
3. Order for $ or above and get X

**Comment by QuickBasket** [comment]
if you ate using WooCommerce please support this request:

⚡️Request for Woo Discount Rules plugin⚡️ #5405
